gradle/libs.versions.toml

```diff
@@ -10,11 +16,12 @@ managed-aws-lambda = '1.2.2'
 managed-aws-lambda-events = '3.11.0'
 managed-aws-serverless-core = '1.9.1'
 managed-aws-cdk-lib='2.58.1'
+managed-micronaut-starter = "3.8.0"
```

Comment:
issue: potential conflict

I am wondering about this dependency. If feels wrong that a Micronaut project "manages" another one. If multiple projects needs micronaut starter, then it would potentially mean a conflict when building the platform BOM, since we could have different managed versions for the same project.

This should probably be replaced with importing the Micronaut Starter catalog in settings....... once it exists! In the meantime, I think we should probably remove the managed flag for this one.

Reply:
I removed the `managed` https://github.com/micronaut-projects/micronaut-aws/pull/1582/commits/873a74b7901a0b5109fc1e1fee63429b09b821ae 

I think the long term solution is that we should treat starter core as a new module.